src/composables/accounts.ts

```diff
@@ -39,12 +59,10 @@ const {
 } = createCallbackRegistry<(newAccount: IAccount, oldAccount: IAccount) => any>();
 
 const areAccountsRestored = ref(false);
+const isMnemonicRestored = ref(false);
+const passwordKey = ref<IKey | null>(null);
```

Comment:
```suggestion
const passwordKey = ref<IKey>();
```

Reply:
`passwordKey` (now `encryptedData`) can also be `null`